assignments/prototypes.js

```diff
@@ -23,6 +33,22 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+  function CharacterStats(charecterAttrs){
+    GameObject.call(this, charecterAttrs);
+    this.thisthis = charecterAttrs.thisthis;
```

Comment:
There should be `healthPoints` and `name` attributes for CharacterStats along with take Damage().  Try to not change naming.